Code/GraphMol/FileParsers/XYZFileParser.cpp

```diff
@@ -0,0 +1,177 @@
+#include <boost/lexical_cast.hpp>
+
+#include "FileParsers.h"
+#include "FileParserUtils.h"
+#include <RDGeneral/StreamOps.h>
+
+#include <RDGeneral/FileParseException.h>
+#include <RDGeneral/BadFileException.h>
+#include <exception>
+
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <string_view>
+#include <cstdlib>
+#include <cstdio>
+
+namespace RDKit {
+
+void ParseExtraLine(std::string_view extraLine) {
+    std::string_view whitespace{" \v\t"};
```

Comment:
huh, I've never seen `\v` before.

Reply:
Yeah I found '\v' in a list of whitespace characters, and thought to include it since it wasn't a newline character --> I searched it up though and it appears to be kind of an outdated character that wouldn't make for an accurate xyz file line anyway